week_1.py

```diff
@@ -0,0 +1,188 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+class CashRegister:
+
+    def __init__(self):
+
+        self.total_items = dict() #None # {'item': 'price'}
+        self.total_price = 0
+        self.discount = 0 #in terms of percent
+
+    def add_item(self, item, price):
+        self.total_items[item] = price
+
+    def remove_item(self, item):
+        self.total_items.pop(item)
+        return self.total_items
+
+    def apply_discount(self, discount, item): #applied discount to the items
+        discount_item = int(self.total_items[item] * ((100-discount)/100))
+        self.total_items[item] = discount_item
+        self.show_items()
```

Comment:
you've interpreted it as a discount on a specific item, but it would be easier to apply a discount at the end to the whole receipt total. It's fine either way, just that it would have been easier to do it the receipt total way.